proposals/20191030-api.md

```diff
@@ -0,0 +1,118 @@
+# Falco APIs
+
+## Summary
+
+This is a proposal to better structure the Falco API.
```

Comment:
I think we should spend a few minutes talking about what exactly the Falco API is. As well as highlighting the problem(s) that it would potentially be solving.

I know we go into detail later, but a sentence or two here might be useful.